tests/trace/test_feedback.py

```diff
@@ -62,3 +68,68 @@ def test_custom_feedback(client) -> None:
 
     with pytest.raises(ValueError):
         trace_object.feedback.add("wandb.trying_to_use_reserved_prefix", value=1)
+
+
+def test_feedback_replace(client) -> None:
+    # Create initial feedback
+    create_req = FeedbackCreateReq(
+        project_id="test/project",
+        weave_ref="weave:///test/project/obj/123:abc",
+        feedback_type="reaction",
+        payload={"emoji": "👍"},
+    )
+    initial_feedback = client.server.feedback_create(create_req)
+
+    # Create another feedback with different type
+    note_feedback = client.server.feedback_create(
+        FeedbackCreateReq(
+            project_id="test/project",
+            weave_ref="weave:///test/project/obj/456:def",
+            feedback_type="note",
+            payload={"note": "This is a test note"},
+        )
+    )
+
+    # Replace the first feedback with new content
+    replace_req = FeedbackReplaceReq(
+        project_id="test/project",
+        weave_ref="weave:///test/project/obj/123:abc",
+        feedback_type="note",
+        payload={"note": "Updated feedback"},
+        feedback_id=initial_feedback.id,
+    )
+    replaced_feedback = client.server.feedback_replace(replace_req)
+
+    # Verify the replacement
+    assert replaced_feedback.id == initial_feedback.id
```

Comment:
this strikes me as odd - that it is the same underlying id. Can you share why you did not generate a new id?

Reply:
i would expect this to be the opposite - the new feedback id should explicitly NOT be the original